src/main/resources/forge_at.cfg

```diff
@@ -157,6 +157,11 @@ public net.minecraft.client.renderer.WorldRenderer func_78909_a(I)I # getColorIn
 public net.minecraft.client.renderer.WorldRenderer func_178972_a(IIIII)V # putColorRGBA
 # ModelBlock Constructor
 public net.minecraft.client.renderer.block.model.ModelBlock <init>(Lnet/minecraft/util/ResourceLocation;Ljava/util/List;Ljava/util/Map;ZZLnet/minecraft/client/renderer/block/model/ItemCameraTransforms;)V
+# RenderManager
+public net.minecraft.client.renderer.entity.RenderManager field_178636_l # skinMap
```

Comment:
Then add a getter, not a "let's let anyone replace this wholesale willy nilly without informing anyone else"